src/Knp/Menu/Renderer/ListRenderer.php

```diff
@@ -45,7 +45,7 @@ public function __construct(MatcherInterface $matcher, array $defaultOptions = [
     }
 
     /**
-     * @param array<string, mixed> $options
+     * @param array<string, string|bool|null> $options
```

Comment:
please be careful when applying requested changes. I requested that for the `$attributes` argument of `renderList`, not for `$options` (where it is wrong. Options are indeed mixed)